components/PicklistBase/PicklistBase.vue

```diff
@@ -25,7 +25,8 @@
                 variant="primary"
                 v-on:click="handleAllButton()"
               >
-                All
+                <span v-if="allPicked">🚫 Clear All</span>
+                <span v-else>✅ Select All</span>
```

Comment:
Leave the text here as just "All".  It is a little less clear, but also takes up less screen real estate. We are designing to run in portrait mode on small mobile (iPhone SE in devtools). Using less text here allows more room for the other columns.